Fix svnlook_tests.py, so it sets PATH in the python hook.

From: David Rothenberger <d.rothenberger@entomo.com>


---
 subversion/tests/cmdline/svnlook_tests.py |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/subversion/tests/cmdline/svnlook_tests.py b/subversion/tests/cmdline/svnlook_tests.py
index 4315168..7ad979b 100755
--- a/subversion/tests/cmdline/svnlook_tests.py
+++ b/subversion/tests/cmdline/svnlook_tests.py
@@ -639,7 +639,7 @@ svnlook_bin=%s
 fp = open(os.path.join(sys.argv[1], 'hooks.log'), 'wb')
 def output_command(fp, cmd, opt):
   command = [svnlook_bin, cmd, '-t', sys.argv[2], sys.argv[1]] + opt
-  process = subprocess.Popen(command, stdin=subprocess.PIPE, stdout=subprocess.PIPE, stderr=subprocess.PIPE, shell=False, bufsize=-1)
+  process = subprocess.Popen(command, stdin=subprocess.PIPE, stdout=subprocess.PIPE, stderr=subprocess.PIPE, shell=False, bufsize=-1, env={'PATH': '/usr/bin'})
   (output, errors) = process.communicate()
   status = process.returncode
   fp.write(output)
